src/modules/VmMigrationDetails/dynamic-plugin.ts

```diff
@@ -0,0 +1,20 @@
+import { EncodedExtension } from '@openshift/dynamic-plugin-sdk';
+import { RoutePage } from '@openshift-console/dynamic-plugin-sdk';
+import type { ConsolePluginMetadata } from '@openshift-console/dynamic-plugin-sdk-webpack/lib/schema/plugin-package';
+
+export const exposedModules: ConsolePluginMetadata['exposedModules'] = {
+  VMMigrationDetails: './modules/VmMigrationDetails/VMMigrationDetailsWrapper',
+};
+
+export const extensions: EncodedExtension[] = [
+  {
+    type: 'console.page/route',
+    properties: {
+      component: {
+        $codeRef: 'VMMigrationDetails',
+      },
+      path: '/mtv/plans/:planName',
```

Comment:
It looks like it will look nice in the plans module, we can move it there in later time